node.sh

```diff
@@ -0,0 +1,53 @@
+package: node
+version: "v9.8.0"
+build_requires:
+  - curl
+prefer_system: "(?!slc5)"
+prefer_system_check: |
+  which node || { echo "node is missing"; exit 1; }
+  if [ `$(brew --prefix node)/bin/node --version | tr -d v | awk -F \. {'print substr(0 $1, length($1), length($1) + 1) substr(0 $2, length($2), length($2) + 1)'}` -le 0809 ]; then echo "higher node version required"; exit 1; fi
```

Comment:
We use bash for executing checks, so use `[[` instead of `[`. Alternatively, to the check you have, you could use:

```bash
[[ printf "%04d%04d%04d" $(node --version | sed -e 's/v//g; s/\./ /g') -le 000900090000 ]]
```

but what you've done is OK.

Reply:
If I use `[[` the number which start from `0809 ` is treated as octal.

Reply:
done